common/law_groups/00_laws.txt

```diff
@@ -0,0 +1,200 @@
+﻿# base_enactment_days = the length of the enactment cycle (in days) prior to taking Legitimacy into account. If not specified, this value will be dictated by the LAW_ENACTMENT_DEFAULT_BASE_DAYS define. If specified, this overrides the define.
+# enactment_approval_mult = multiplier to the approval effects changing this law has on IGs, default 1.0
+
+# enable = {} trigger for whether laws in this group should be possible at all
+
+lawgroup_governance_principles = {
+	law_group_category = power_structure
+	base_enactment_days = 200
+	enactment_approval_mult = 2.0
+	
+	progressive_movement_chance = 1
+	regressive_movement_chance = 0.25
+	
+	change_allowed_trigger = {
+		NAND = {
+			is_subject = yes
+			OR = {
+				is_subject_type = subject_type_personal_union
+				has_government_type = gov_chartered_company
+			}			
+		}
+	}
+}
+
+lawgroup_distribution_of_power = {
```

Comment:
Maybe also here keep the original values in comments, I don't know how much work this is. But specifically for enactment)days and approval, it would be nice to have the vanilla values in there.

Reply:
I'll update both sections with those before asking for a second review, yea. I think I just forgot to do it while implementing all the changes.